remote-access/samba.md

```diff
@@ -0,0 +1,99 @@
+## Samba/CIFS
+
+Samba is a implementation of the SMB/CIFS networking protocol which is used by Windows devices to providing shared access to files, printers, and serial ports etc. There is a comprehensive [Wikipedia page](https://en.wikipedia.org/wiki/Samba_(software)) which describes Samba and its capabilities.
+
+This page will explain how to use a subset of the Samba system to 'mount' a shared folder on a Windows device so it appears on your Raspberry Pi, or to share a folder on your Raspberry Pi so it can be accessed by a Windows Client.
+
+### Installing CIFS/Samba support
+
+By default, Raspbian does not include CIFS/Samba support, but it can easily be added. The following commmand will install all the required components for using Samba as a server or a client.
+```
+sudo apt-get update
+sudo apt-get install -y samba samba-common-bin smbclient cifs-utils
+```
+
+### Using a shared Windows folder
+
+First, you need to share a folder on your windows device. This is quite a convoluted process!
+
+#### Turn on sharing
+
+1. Open the Networking and Sharing Centre by right clicking on the system tray and selecting.
+2. Click on `Change advanced sharing settings`
+3. Select Turn on network discovery
+4. Select Turn on  file and printer sharing
+5. Save changes
+
+#### Share the folder
```

Comment:
I can't tell if these headers are supposed to be nested or not - but this is H4 which is quite granular